src/operation.rs

```diff
@@ -805,7 +805,7 @@ impl Validate {
                 Ok(validity::RequestList::single(prefix, asn))
             }
             ValidateWhat::File(ref path) => {
-                let mut file = match fs::File::open(path) {
+                let file = match fs::File::open(path) {
                     Ok(file) => file,
```

Comment:
Since we don’t need `file` to be a `File` further down, you can just wrap it in a `BufReader` right here in the Ok branch.

(Same also further down.)

Reply:
alright, now fixed!